components/script/script_runtime.rs

```diff
@@ -776,3 +780,44 @@ impl Deref for JSContext {
         &self.0
     }
 }
+
+pub struct StreamConsumer(*mut JSStreamConsumer);
+
+#[allow(unsafe_code)]
+impl StreamConsumer {
+    fn consumeChunk(sc: &mut JSStreamConsumer, begin: *const u8, length: usize) -> bool {
+        unsafe {
+            return StreamConsumerConsumeChunk(sc, begin, length);
+        }
+    }
+
+    fn streamEnd(sc: &mut JSStreamConsumer){
```

Comment:
Let's give these methods `snake_case` names instead of `camelCase` ones.

Reply:
Done